src/shared/actions/page/communities/iot/assets.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Actions powering the UI state of Iot community's assets page.
+ */
+
+import { createActions } from 'redux-actions';
+
+/**
+ * toggle grid display mode
+ * */
+function toggleGrid() {
+  return 'grid';
+}
+
+/**
+ * toggle list display mode
+ * */
+function toggleList() {
+  return 'list';
+}
```

Comment:
Bad design. Should be:
- 1 variable in the Redux store keeps the current view mode (`grid`, `list`, etc.)
- A map exported from this actions module gives all valid values for that variable.
- A single action `toggleView(mode)` switches page into the desired view.